.github/main.yml

```diff
@@ -0,0 +1,22 @@
+name: GitHub Actions Demo
+on: [push]
+jobs:
+  Test-With-GitHub-Actions:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out repository code
+        uses: actions/checkout@v3
+      - name: Install nodejs
+        uses: actions/setup-node@v3
+        with:
+          node-version: '16'
+      - name: Install cypress
+        run: npm install cypress@latest
+      - name: Install linter
+        run: npm install eslint --save-dev
+      - name: Install linter cypress
+        run: npm install eslint-plugin-cypress --save-dev
```

Comment:
these installs are not needed, as the dependencies are already specified in package.json file, the only required step is to run the ci:
```
      - name: Install
        run: npm ci
```